lib/ansible/modules/cloud/amazon/elb_application_lb.py

```diff
@@ -119,6 +119,12 @@
     description:
       - The time in seconds to use in conjunction with I(wait).
     version_added: 2.6
+  purge_rules:
+    description:
+      - Keeps the existing load balancer rules in place. Will modify and add, but will not delete.
```

Comment:
This description reads as if setting `purge_rules` to `yes` would keep the existing load balancer rules in place. 

Perhaps just prefix it with 'When set to no, keep the existing...'